setup.cfg

```diff
@@ -60,18 +58,11 @@ upload_dir = docs/_build/html
 show_response = 1
 
 [tool:pytest]
```

Comment:
Maybe we should migrate this configuration to `[tool.pytest.ini_options]` under `pyproject.toml`, as noted [here](https://docs.pytest.org/en/6.2.x/customize.html#pyproject-toml) and [here](https://docs.pytest.org/en/6.2.x/customize.html#setup-cfg).

Reply:
Ah that makes sense. In that case, everything else looks good to me (except for the pickle file, which we can come back to in the future anyway).